blog/templates/blog/view_entry.html

```diff
@@ -0,0 +1,6 @@
+{% extends "blog/blogbase.html" %}
+
+{% block blog_content %}
+    <h2>{{ entry.title }}</h2>
+    <p>{{ entry.body }}</p>
```

Comment:
Add additional fields here.  For example, you store the date the entry was created.  It's useful for an end user to know when the blog entry was written.  Add the created_dttm to the view.